mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/AvtaleRoutes.kt

```diff
@@ -168,4 +168,5 @@ data class AvtaleRequest(
     val personopplysninger: List<Personopplysning>,
     val personvernBekreftet: Boolean,
     val nusData: JsonObject?,
```

Comment:
Kommentar litt utenfor scope av denne PRen, men er det en grunn til at denne egenskapen er `JsonObject` i stedet for `NusData`? Det burde vel holde å gjøre serialisering/deserialisering ved sql-spørringene.

Ser ut til å gjelde `AvtaleRequest`, `AvtaleDbo`, `TiltaksgjennomforingRequest` og `TiltaksgjennomforingDbo`.

Reply:
Lurte på det samme, tror vi bare kan endre de ja